cdk/bin/cdk.ts

```diff
@@ -38,3 +40,35 @@ new SloMonitoring(app, 'SloMonitor-PROD', {
 	stack: 'mobile-notifications',
 	stage: 'PROD',
 });
+
+export const senderCodeProps = {
+	appName: 'sender-worker',
+	stack: 'mobile-notifications-workers',
+	stage: 'CODE',
+	asgCapacity: {
+		minimumInstances: 1,
```

Comment:
during the testing phase, would we consider creating code and prod with the same settings?

Reply:
Yes, that sounds good for the testing phase.